tests/test-basic-user-only.sh

```diff
@@ -112,3 +112,15 @@ $OSTREE checkout --force-copy perms out
 $OSTREE checkout ${CHECKOUT_H_ARGS} --union-identical perms out
 $OSTREE fsck
 echo "ok checkout checksum with canonical perms"
+
+cd ${test_tmpdir}
+rm repo -rf
+ostree_repo_init repo init --mode=bare-user-only
+rm files -rf && mkdir files
+echo afile > files/afile
+$OSTREE commit --no-xattrs -b perms files
```

Comment:
We should be able to drop `--canonical-permissions` from `COMMIT_ARGS` and use `COMMIT_ARGS` here too, no?

Reply:
Yes! And... it uncovered another bug :disappointed:
It looks like file adoption (`commit --consume`) is also broken for bare-user-only.